packages/icons/src/companies/bolt-eu-icon.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+/**
+ * For the company https://bolt.eu/
```

Comment:
If we have multiple Bolt companies, it's probably better to err on the side of more info vs. less info in this comment.

Reply:
FWIW, Bolt no longer operates in Portland.  This could change again in the future, but we personally (regionally?) don't need no stinkin' Bolt icon.

Reply:
What other info should I include, @landonreed?

Reply:
I was just thinking mentioning the mode that Bolt operates.